adapters/networkHandlerFactory.js

```diff
@@ -7,11 +7,15 @@ const {
 const {
   networkHandler: GenericNetworkHandler
 } = require("./networkhandler/genericNetworkHandler");
+const {
+  networkHandler: GoogleAdwordsRemaketingListNetwoekHandler
+} = require("../v0/destinations/google_adwords_remarketing_lists/util");
 
 const handler = {
   generic: GenericNetworkHandler,
   braze: BrazeNetworkHandler,
-  marketo: MarketoNetworkHandler
+  marketo: MarketoNetworkHandler,
+  google_adwords_remarketing_lists: GoogleAdwordsRemaketingListNetwoekHandler
```

Comment:
```suggestion
  google_adwords_remarketing_lists: GoogleAdwordsRemarketingListNetworkHandler
```
same applies for here